src/main/resources/view/HelpWindow.fxml

```diff
@@ -19,27 +23,48 @@
         <URL value="@Fonts.css" />
         <URL value="@HelpWindow.css" />
       </stylesheets>
-
-      <HBox alignment="CENTER">
-        <children>
-          <Label fx:id="helpMessage" text="Label">
-            <HBox.margin>
-              <Insets right="5.0" />
-            </HBox.margin>
-          </Label>
-          <Button fx:id="copyButton" mnemonicParsing="false" onAction="#copyUrl" text="Copy URL">
-            <HBox.margin>
-              <Insets left="5.0" />
-            </HBox.margin>
-          </Button>
-        </children>
-        <opaqueInsets>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </opaqueInsets>
-        <padding>
-          <Insets bottom="10.0" left="5.0" right="10.0" top="5.0" />
-        </padding>
-      </HBox>
+         <VBox minWidth="550.0">
+            <children>
+               <TableView fx:id="table" fixedCellSize="0.0" layoutY="-1.0" minHeight="500.0" minWidth="550.0" VBox.vgrow="ALWAYS">
+                 <columns>
+                   <TableColumn fx:id="action" prefWidth="111.0" reorderable="false" text="Action" />
+                   <TableColumn fx:id="format" prefWidth="232.0" reorderable="false" text="Format" />
+                     <TableColumn fx:id="example" prefWidth="229.0" reorderable="false" text="Example" />
+                 </columns>
+                  <opaqueInsets>
+                     <Insets />
+                  </opaqueInsets>
+               </TableView>
+            <HBox alignment="CENTER" layoutX="1.0" layoutY="536.0" minHeight="80.0" minWidth="550.0" style="-fx-background-color: D3DAFB50;">
+              <children>
+                <Label fx:id="helpMessage" text="Label" textFill="#424874" wrapText="true" HBox.hgrow="ALWAYS">
```

Comment:
not so important but the padding missing when minimised
![image](https://user-images.githubusercontent.com/35279431/136804061-952cd246-e111-4d9a-a1c8-56bcb19a6a93.png)

Also just my opinion but the footer can have different color to differentiate from the table.
The scroll bar a bit hard to see as well.

Reply:
Padding issue not observed on my side as well. I prefer a more visible scroll bar as well but not a major issue.